### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

**Comment by Zmoki** [comment]
[Widget & CKEditor: обновить](https://trello.com/c/alYu5WGU/216-widget-ckeditor-%D0%BE%D0%B1%D0%BD%D0%BE%D0%B2%D0%B8%D1%82%D1%8C)

File: README.md [hunk 1]
```diff
@@ -41,51 +52,56 @@ CKEDITOR.plugins.addExternal('uploadcare', '/absolute/path/to/uploadcare/plugin.
 
 You can find more information about how to manually install plugins on [CKEditor Documentation][ck-docs-manual-install].
```
Comment (dayton1987) [inline README.md]: You can find more info on manually installing plugins in the CKEditor documentation.

File: README.md [hunk 2]
```diff
@@ -41,51 +52,56 @@ CKEDITOR.plugins.addExternal('uploadcare', '/absolute/path/to/uploadcare/plugin.
...
 
-## Configure
+## Usage
 
-Initialize a CKEditor plugin with additional params:
+Add `uploadcare` to the list of CKEditor plugins and toolbar.
```
Comment (dayton1987) [inline README.md]: ... to your list of CKEditor plugins and toolbar.

File: README.md [hunk 3]
```diff
@@ -41,51 +52,56 @@ CKEDITOR.plugins.addExternal('uploadcare', '/absolute/path/to/uploadcare/plugin.
...
+})
+```
 
-1. Press "Uploadcare" button.
-2. Select a file to upload and press "Upload"
-3. Wait for file to be uploaded.
-4. Crop an image as you wish.
-5. Click "Done". A cropped image will be available inside editor.
+You can heavily customize widget behavior, i.e. file sources, file validation and much more, please
+read [Uploadcare Widget][widget-docs-config] and [JavaScript API][widget-docs-js-api] documentation.
```
Comment (dayton1987) [inline README.md]: You can deeply customize the widget behavior: file sources, file validation, and much more. Please, check out the Uploadcare Widget and JavaScript API documentation.

File: README.md [hunk 4]
```diff
@@ -41,51 +52,56 @@ CKEDITOR.plugins.addExternal('uploadcare', '/absolute/path/to/uploadcare/plugin.
...
 
 ## Feedback
 
 Send any feedback or request support at hello@uploadcare.com
```
Comment (dayton1987) [inline README.md]:
```
Your feedback or support requests are welcome at [hello@uploadcare.com][uc-email-hello].

[uc-email-hello]: mailto:hello@uploadcare.com
```

**Comment by Zmoki** [comment]
@dayton1987 bump

File: README.md [hunk 5]
```diff
@@ -1,29 +1,40 @@
...
 
 ## Install
 
+You’re free to choose from the install methods listed below.
+
 ### Automatic install
-Simply add [Uploadcare plugin](http://ckeditor.com/addon/uploadcare) to [CKBuilder][ck-docs-online-builder].
 
-You can find more inforamtion at [CKBuilder Documentation][ck-docs-auto-install].
+Simply add the [File Uploader plugin](http://ckeditor.com/addon/uploadcare) to [CKBuilder][ck-docs-online-builder].
```
Comment (nd0ut) [inline README.md]: ссылку заалиасить бы

**Pull request merged**